stack.yaml

```diff
@@ -1,13 +1,13 @@
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-prelude/c40175657bc8134e0561d9b6e6334728c19112e4/snapshot.yaml
+resolver: lts-14.22
```

Comment:
that's not an option. I thought we have to stick to 'cardano-prelude'.

Reply:
That snuck in during rebase, reverting..